src/chromo/common.py

```diff
@@ -708,6 +713,30 @@ def kinematics(self, kin):
         self._kinematics = kin
         self._set_kinematics(kin)
 
+    @property
+    def final_state_particles(self):
+        """Returns sorted list"""
+        uids = np.unique(self._final_state_particles)
+        return list(uids[np.argsort(2 * np.abs(uids) - (uids < 0))])
```

Comment:
I said it return a list, but after more thought it should be a tuple instead.

Reply:
> It is model specific.

I checked all models what they decay and what not. It turns out that only EposLHC decays everything that could be passed to it as unstable particles. Models that use Pythia6 (Sibyll, Sophia) and Pythia6 itself cannot decay neutron. Phojet, Dpmjet - just not decay what set as unstable. UrQMD and QGSJet just don't decay particles.

So, I would say we need DecayHandler for pretty much everything.